Mathlib/Probability/Variance.lean

```diff
@@ -223,12 +216,13 @@ theorem variance_le_expectation_sq [IsProbabilityMeasure μ] {X : Ω → ℝ}
   · exact (AEMeasurable.pow_const (hm.aemeasurable.sub_const _) _).aestronglyMeasurable
 
 theorem evariance_def' [IsProbabilityMeasure μ] {X : Ω → ℝ} (hX : AEStronglyMeasurable X μ) :
-    evariance X μ= (∫⁻ ω, (‖X ω‖ₑ ∂μ) - ENNReal.ofReal (μ[X] ^ 2) := by
+    evariance X μ = (∫⁻ ω, ‖X ω‖ₑ ^ 2 ∂μ) - ENNReal.ofReal (μ[X] ^ 2) := by
```

Comment:
Ah, so a previous commit was wrong? Let me flag this (for myself) for checking the overall diff.

Reply:
Checked: that diff looks fine.